### PR TITLE
Core/Security:check for fake packets and disaster addons what make client crash

### PR DESCRIPTION
**Pull request by Viste**
##### CHANGES PROPOSED:

- add check for fake packets and disaster addons what make client crash

Closes: no issue for this 

to test  try crash client with some addon or fake packet 

##### Target branch(es):Master

### CODE REVIEW

File: src/server/game/Handlers/ArenaTeamHandler.cpp [hunk 1]
```diff
@@ -145,6 +146,11 @@ void WorldSession::HandleArenaTeamInviteOpcode(WorldPacket & recvData)
         return;
     }
 
+    //check for fake packets and disaster addons what make client crash
```
Comment (FrancescoBorzi) [inline src/server/game/Handlers/ArenaTeamHandler.cpp]: please reword in `// check for fake packets and bad addons that cause client to crash`

**Comment by Viste** [comment]
>_< commits from work account what created for self-hosted gitlab >_>

File: src/server/game/Handlers/ChatHandler.cpp [hunk 2]
```diff
@@ -315,6 +315,24 @@ void WorldSession::HandleMessagechatOpcode(WorldPacket & recvData)
     }
 
 
+    // GS Reguest — another disaster addons what make client crash
```
Comment (FrancescoBorzi) [inline src/server/game/Handlers/ChatHandler.cpp]: please reword in `// GS Reguest — another check for broken addons that cause client crash`

File: src/server/game/Handlers/MailHandler.cpp [hunk 3]
```diff
@@ -187,6 +188,10 @@ void WorldSession::HandleSendMail(WorldPacket & recvData)
         return;
     }
 
+    //check for fake packets and disaster addons what make client crash
```
Comment (FrancescoBorzi) [inline src/server/game/Handlers/MailHandler.cpp]: same as above

**Comment by FrancescoBorzi** [comment]
@Viste you can fix it by squashing your commits and then chaning the author: https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit

**Comment by BarbzYHOOL** [comment]
if you need help with "advanced" operations with git, you can ask me in discord, I can try to help

**Pull request closed without merging**